docs/source/FPGA-to-bitstream/Yosys compilation.rst

```diff
@@ -23,10 +23,8 @@ calling the following command:
 .. code-block:: console
 
         # Nextpnr backend synthesis (JSON)
-        synthesis_npnr <path_to_user_design>
+        synthesis <path_to_user_design>
```

Comment:
Is the change to the CLI code also made in this PR? Couldn't see it at a quick skim, sorry if I've missed it